tests/api_test.py

```diff
@@ -159,6 +159,28 @@ def test_create_newdle_participant_email_sending(flask_client, dummy_uid, mail_q
     assert resp.status_code == 200
 
 
+def test_get_busy_times(flask_client, dummy_uid, monkeypatch):
+    def mock_get_busy(date, tz, uid):
+        return {}
+
+    from newdle import api
+
+    monkeypatch.setattr(api, '_get_busy_times', mock_get_busy)
```

Comment:
```python
monkeypatch.setattr(api, '_get_busy_times', lambda date, tz, uid: {})
```

Or better, use a real mock - that will let you actually assert that it has been called with the proper data from the request! That way the test also becomes more useful than just checking if the access check works.

You can either use `monkeypatch.setattr` for it or better add [`pytest-mock`](https://pypi.org/project/pytest-mock/) to the dev dependencies and use that here.

Reply:
Thanks again! I'll get on it